src/components/Navbar/Navbar.jsx

```diff
@@ -5,13 +5,13 @@ import { paths } from '../../services/path';
 export const Navbar = () => {
   return (
     <Box py={4}>
-      <Container maxW="container.xl">
-        <HStack justifyContent="space-between">
-          <Heading size="md" as={Link} href={paths.home}>
+      <Container maxW="container.xl" >
+        <HStack justifyContent="space-between" h="50px" bg="#f6f6f6f6" >
+          <Heading p={4} color="#0f9d58" size="md" style={{textDecoration: 'none'}} as={Link} href={paths.home}>
```

Comment:
## color="#0f9d58"

It's usually fine for personal projects, but usually we try to not hard code colors like this. Can you try something like
```
<Heading p={4} color="green.500" {...} />
```
See
https://chakra-ui.com/docs/styled-system/css-variables
https://chakra-ui.com/docs/styled-system/theme

## style={{textDecoration: 'none'}}
In general, if you are using a React component library, don't use `style` unless you have to. If you do need to override something, use `sx` instead (https://chakra-ui.com/docs/styled-system/the-sx-prop).

I believe `Heading` already have `textDecoration` props, so you should use that.